File: Respostas/Bee_1048/1048.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) throws IOException {
		double salario;
		
		Scanner input = new Scanner(System.in);
		
		salario = input.nextDouble();
		
		if (salario > 0 && salario <= 400) {
			System.out.printf("Novo salario: %.2f\n", (salario * 0.15)  + salario);
			System.out.printf("Reajuste ganho: %.2f\n", salario * 0.15);
			System.out.print("Em percentual: 15 %\n");
		}
		
		else if (salario >= 400.01 && salario <= 800) {
			System.out.printf("Novo salario: %.2f\n", (salario * 0.12) + salario);
			System.out.printf("Reajuste ganho: %.2f\n", salario * 0.12);
			System.out.printf("Em percentual: 12 %\n");
		}
		
		else if (salario >= 800.01 & salario <= 1200) {
			System.out.printf("Novo salario: %.2f\n", (salario * 0.10) + salario);
			System.out.printf("Reajuste ganho: %.2f\n", salario * 0.10);
			System.out.print("Em percentual: 10 %\n");
		}
		
		else if (salario >= 1200.01 & salario <= 2000) {
			System.out.printf("Novo salario: %.2f\n", (salario * 0.07) + salario);
			System.out.printf("Reajuste ganho: %.2f\n", salario * 0.07);
			System.out.print("Em percentual: 7 %\n");
		}
		
		else if (salario > 2000) {
			System.out.printf("Novo salario: %.2f\n", (salario * 0.04) + salario);
			System.out.printf("Reajuste ganho: %.2f\n", salario * 0.04);
			System.out.print("Em percentual: 4 %\n");
		}
	}
}

File: Respostas/Bee_1048/1048.c
#include <stdio.h>

int main() {
	double salario;
	
	scanf("%lf", &salario);
	
	if (salario > 0 && salario <= 400) {
		printf("Novo salario: %.2lf\n", (salario * 0.15)  + salario);
		printf("Reajuste ganho: %.2lf\n", salario * 0.15);
		printf("Em percentual: 15 %%\n");
	}
	
	else if (salario >= 400.01 && salario <= 800) {
		printf("Novo salario: %.2lf\n", (salario * 0.12) + salario);
		printf("Reajuste ganho: %.2lf\n", salario * 0.12);
		printf("Em percentual: 12 %%\n");
	}
		
	else if (salario >= 800.01 & salario <= 1200) {
		printf("Novo salario: %.2lf\n", (salario * 0.10) + salario);
		printf("Reajuste ganho: %.2lf\n", salario * 0.10);
		printf("Em percentual: 10 %%\n");
	}
			
	else if (salario >= 1200.01 & salario <= 2000) {
		printf("Novo salario: %.2lf\n", (salario * 0.07) + salario);
		printf("Reajuste ganho: %.2lf\n", salario * 0.07);
		printf("Em percentual: 7 %%\n");
	}
		
	else if (salario > 2000) {
		printf("Novo salario: %.2lf\n", (salario * 0.04) + salario);
		printf("Reajuste ganho: %.2lf\n", salario * 0.04);
		printf("Em percentual: 4 %%\n");
	}
	
	return 0;
}